scompose/project/instance.py

```diff
@@ -152,6 +152,10 @@ def set_network(self, params):
         if "allocate_ip" not in self.network:
             self.network["allocate_ip"] = True
 
+        # if not specified, set the default value for enable property
+        if "enable" not in self.network:
+            self.network["enable"] = True
```

Comment:
For these two, how about:
```python
for key in ["enable", "allocate_ip"]:
    self.network[key] = self.network.get(key, True)
```

Reply:
I like the suggestion! Will do it it now